.gitignore

```diff
@@ -1,5 +1,4 @@
 /sequins
-/sequins-dump
```

Comment:
ok! I only ask because I generally try to make unrelated changes as separate commits for compartmentalization purposes. But that's just me! :D